packages/cli/src/commands/__tests__/commands.spec.ts

```diff
@@ -0,0 +1,140 @@
+    await new Promise(resolve => {
+      parser.parse(`${command} /path/to ${upstream}`, (_err: Error, commandPromise: Promise<unknown>) =>
+        commandPromise.then(resolve),
+      );
+    });
+
+    expect(createMultiProcessPrism).not.toHaveBeenCalled();
+    expect(createSingleProcessPrism).toHaveBeenLastCalledWith({
+      operations: [],
+      multiprocess: false,
+      log: 'stdout',
+      cors: true,
+      host: '127.0.0.1',
+      port: 4010,
+      ...additionalCallOptions,
+    });
+  });
+
+  test(`starts ${command} server on custom port`, async () => {
+    await new Promise(resolve => {
+      parser.parse(`${command} /path/to -p 666 ${upstream}`, (_err: Error, commandPromise: Promise<unknown>) =>
+        commandPromise.then(resolve),
+      );
+    });
+
+    expect(createMultiProcessPrism).not.toHaveBeenCalled();
+    expect(createSingleProcessPrism).toHaveBeenLastCalledWith({
+      operations: [],
+      multiprocess: false,
+      log: 'stdout',
+      cors: true,
+      host: '127.0.0.1',
+      port: 666,
+      ...additionalCallOptions,
+    });
+  });
+
+  test(`starts ${command} server on custom host`, async () => {
+    await new Promise(resolve => {
+      parser.parse(`${command} /path/to -h 0.0.0.0 ${upstream}`, (_err: Error, commandPromise: Promise<unknown>) =>
+        commandPromise.then(resolve),
+      );
+    });
+
+    expect(createMultiProcessPrism).not.toHaveBeenCalled();
+    expect(createSingleProcessPrism).toHaveBeenLastCalledWith({
+      operations: [],
+      multiprocess: false,
+      log: 'stdout',
+      cors: true,
+      host: '0.0.0.0',
+      port: 4010,
+      ...additionalCallOptions,
+    });
+  });
+
+  test(`starts ${command} server on custom host and port`, async () => {
+    await new Promise(resolve => {
+      parser.parse(
+        `${command} /path/to -p 666 -h 0.0.0.0 ${upstream}`,
+        (_err: Error, commandPromise: Promise<unknown>) => commandPromise.then(resolve),
+      );
+    });
+
+    expect(createMultiProcessPrism).not.toHaveBeenCalled();
+    expect(createSingleProcessPrism).toHaveBeenLastCalledWith({
+      operations: [],
+      cors: true,
+      log: 'stdout',
+      multiprocess: false,
+      host: '0.0.0.0',
+      port: 666,
+      ...additionalCallOptions,
+    });
+  });
+
+  test(`starts ${command} server with multiprocess option `, async () => {
+    await new Promise(resolve => {
+      parser.parse(`${command} /path/to -m -h 0.0.0.0 ${upstream}`, (_err: Error, commandPromise: Promise<unknown>) =>
+        commandPromise.then(resolve),
+      );
+    });
+
+    expect(createSingleProcessPrism).not.toHaveBeenCalled();
+    expect(createMultiProcessPrism).toHaveBeenLastCalledWith({
+      operations: [],
+      cors: true,
+      log: 'stdout',
+      multiprocess: true,
+      host: '0.0.0.0',
+      port: 4010,
+      ...additionalCallOptions,
+    });
+  });
+
+  test(`starts ${command} server with httpResponse log option `, async () => {
```

Comment:
```suggestion
  test(`starts ${command} server with httpBody log option `, async () => {
```

Reply:
Not too sure I understand this. Are you proposing to change the option name?

Reply:
@XVincentX, yes. Because a response is comprised of headers and a payload. For now we have `httpHeader` and `httpResponse`. Think we should change `httpResponse` to either `httpBody` or `httpPayload`. If we do this we name the entities correctly. We can think of it as `response` being an aggregate and `header(s)` and `body`, entities it aggregates. WDYT?

Reply:
Sounds reasonable. @philsturgeon What do you think?